dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -71,12 +71,20 @@ export class CustomPanelsAdaptor {
         objectType: 'operationalPanel',
         maxItems: 10000,
       });
-      return response.observabilityObjectList.map((panel: any) => ({
-        name: panel.operationalPanel.name,
-        id: panel.objectId,
-        dateCreated: panel.createdTimeMs,
-        dateModified: panel.lastUpdatedTimeMs,
-      }));
+      const panelList: any[] = [];
+      for (const panel of response.observabilityObjectList) {
+        if (panel.operationalPanel.applicationId) {
+          break;
```

Comment:
do you mean continue? if so this can be done by adding a `.filter` before `.map` in the original code

Reply:
Very smart. Changed to `.filter` in latest commit!